src/silo/prepare_dataset.cpp

```diff
@@ -345,70 +287,60 @@ void sortChunk(
       // Reset gpointer, read file again, putting every genome at the correct position.
       // Write file to ostream
 
-      struct EPIDate {
-         uint64_t epi;
+      struct KeyDatePair {
```

Comment:
It's kinda funny that something called `Pair` has three members ;)

Reply:
I thought about that too, but figured that the position in the file is more "meta info". Also not fully happy with the name, but KeyDate would definitely be worse

Reply:
Ok, if you think it fits, we can leave it that way. At least it already suggests, that the third field is maybe not that important.